frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -180,6 +180,29 @@ action.GitMachete.SlideOutSelectedBranchAction.description=Slide out selected br
 
 # Graph Table
 
+string.GitMachete.BranchOrCommitCellRendererComponent.inferred-fork-point.commit=commit
+string.GitMachete.BranchOrCommitCellRendererComponent.inferred-fork-point.fork-point=fork point ?
+string.GitMachete.BranchOrCommitCellRendererComponent.inferred-fork-point.found-in-reflog=has been found in reflog of
+
+string.GitMachete.BranchOrCommitCellRendererComponent.ongoing-operation.applying=APPLYING
+string.GitMachete.BranchOrCommitCellRendererComponent.ongoing-operation.bisecting=BISECTING
+string.GitMachete.BranchOrCommitCellRendererComponent.ongoing-operation.cherry-picking=CHERRY-PICKING
+string.GitMachete.BranchOrCommitCellRendererComponent.ongoing-operation.merging=MERGING
+string.GitMachete.BranchOrCommitCellRendererComponent.ongoing-operation.rebasing=REBASING
+string.GitMachete.BranchOrCommitCellRendererComponent.ongoing-operation.reverting=REVERTING
+
+string.GitMachete.BranchOrCommitCellRendererComponent.stps-tooltip.in-sync=Branch ''{0}'' is in sync with ''{1}''
+string.GitMachete.BranchOrCommitCellRendererComponent.stps-tooltip.in-sync-but-fork-point-off=Branch ''{0}'' is in sync with ''{1}'', but the range of commits belonging to ''{0}'' is uncertain; consider overriding fork point from right-click menu
+string.GitMachete.BranchOrCommitCellRendererComponent.stps-tooltip.merged-to-parent=Branch ''{0}'' is merged to ''{1}''; consider selecting ''Slide out'' in right-click menu
```

Comment:
`consider sliding it out from...`

Reply:
Yeah... I've commented about that already, it sounds like *it's from the menu itself* that the branch is supposed to be slid out :/